pep-0570.rst

```diff
@@ -176,6 +229,59 @@ This is a well discussed, recurring topic on the Python mailing lists:
 * May 2006: `Benji York: [Python-Dev] Positional-only Arguments
   <https://mail.python.org/pipermail/python-dev/2006-May/064790.html>`_
 
+Positional-only parameters have also the (minor) advantage of enforcing
+some logical order when calling interfaces that make use of them. For
+example, the ``range`` function takes all its parameters positionally and
+this disallows forms like:
+
+    range(stop=5, start=0, step=2)
+    range(stop=5, step=2, start=0)
+    range(step=2, start=0, stop=5)
+    range(step=2, stop=5, start=0)
+
+at the price of disallowing using keyword arguments for the (unique)
+intended order:
+
+    range(start=0, stop=5, step=2)
+
+Another critical aspect that motivates positional-only arguments is
+PEP 399 [#PEP399]_: Pure Python/C Accelerator Module Compatibility
+Requirements.  This PEP states that:
+
+    This PEP requires that in these instances that the C code must pass
+    the test suite used for the pure Python code to act as much as
+    a drop-in replacement as reasonably possible
+
+It is clear that if the C code is implemented using the existing capabilities
+to implement positional-only parameters using the argument clinic and
+relating machinery; then, it is not possible for the pure Python counterpart
+to match the provided interface and requirements. This also creates a disparity
+between the interfaces of some functions and classes in the CPython standard
+library and other Python implementations. For example:
+
+    $ python3 # CPython 3.7.2
+    >>> import binascii; binascii.crc32(data=b'data')
+    TypeError: crc32() takes no keyword arguments
+
+    $ pypy3 # PyPy 6.0.0
+    >>>> import binascii; binascii.crc32(data=b'data')
+    2918445923
+
+Other Python implementations can, of course, reproduce the CPython APIs
+manually, but this goes against the spirit of PEP 399 [#PEP399]_ that
+intends to avoid duplication effort by mandating that all modules added
+to Python's standard library **must** have a pure Python implementation
+with the same interface and semantics.
+
+A final argument in favor of positional-only arguments is that they allow
+some new optimisations like the ones already present in the argument clinic
```

Comment:
"optimisations" is the British spelling. Not sure if there's a style guide or if it recommends BrE or AmE. Probably best to go with style guide if it exists, otherwise just consistency throughout (which, again, I haven't checked for yet :P)